public/docs/_examples/forms/e2e-spec.ts

```diff
@@ -45,9 +46,8 @@ describeIf(browser.appIsTs || browser.appIsJs, 'Forms Tests', function () {
     let test = 'testing 1 2 3';
     let newValue: string;
     let alterEgoEle = element.all(by.css('input[name=alterEgo]')).get(0);
-    alterEgoEle.getAttribute('value').then(function(value) {
-      // alterEgoEle.sendKeys(test);
-      sendKeys(alterEgoEle, test);
+    alterEgoEle.getAttribute('value').then(function(value: string) {
+      alterEgoEle.sendKeys(test);
       newValue = value + test;
       expect(alterEgoEle.getAttribute('value')).toEqual(newValue);
     }).then(function() {
```

Comment:
Suggestion: since the function(value: string) returns void, should this be rewritten as:

```
  alterEgoEle.getAttribute('value').then((value: string): void => {
    alterEgoEle.sendKeys(test);
    newValue = value + test;
    expect(alterEgoEle.getAttribute('value')).toEqual(newValue);
  });
  let b = element.all(by.css('button[type=submit]')).get(0);
  b.click().then((): void => {
    let alterEgoTextEle = element(by.cssContainingText('div', 'Alter Ego'));
    expect(alterEgoTextEle.isPresent()).toBe(true, 'cannot locate "Alter Ego" label');
    let divEle = element(by.cssContainingText('div', newValue));
    expect(divEle.isPresent()).toBe(true, 'cannot locate div with this text: ' + newValue);
  });
```

?